toffy/rename_fovs.py

```diff
@@ -0,0 +1,107 @@
+import json
+import os
+import warnings
+from distutils.dir_util import copy_tree
+
+from ark.utils import io_utils
+from ark.utils.misc_utils import verify_in_list
+from toffy.tiling_utils import rename_duplicate_fovs
+
+
+def rename_missing_fovs(fov_data):
+    """Identify FOVs that are missing the 'name' key and create one with value placeholder_{n}
+    Args:
+        fov_data (dict): the FOV run JSON data
+
+    Returns:
+        dict: a copy of the run JSON data with placeholder names for FOVs that lack one
+       """
+
+    copy_fov_data = fov_data
```

Comment:
Check to confirm if this actually makes a copy, or just a reference to the original. I can never remember with dicts what the default is.